.dagger/docs.go

```diff
@@ -159,7 +160,8 @@ func (d Docs) Generate() *dagger.Directory {
 		WithDirectory("", d.GenerateSchema()).
 		WithDirectory("", d.GenerateCli()).
 		WithDirectory("", d.GenerateSchemaReference()).
-		WithDirectory("", d.GenerateConfigSchemas())
+		WithDirectory("", d.GenerateConfigSchemas()).
+		WithDirectory("", d.GenerateSchemaReference())
```

Comment:
:thinking: `GenerateSchemaReference` is repeated twice in this list, should this be `GenerateReferencePage` instead?